src/DotNetWorker/Http/IHttpRequestData.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Security.Claims;
+using Microsoft.Azure.Functions.Worker.Http;
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    /// <summary>
+    /// A representation of the HTTP request sent by the host.
+    /// </summary>
+    public interface IHttpRequestData
```

Comment:
Do we need this?

Reply:
No. As discussed, this was added to support the approach of augmenting the gRPC types with the implementation, but that wasn't feasible and the interface was left behind. Removed now.